contractcourt/utxonursery.go

```diff
@@ -352,14 +356,14 @@ func (u *UtxoNursery) IncubateOutputs(chanPoint wire.OutPoint,
 	default:
 	}
 
-	numHtlcs := len(incomingHtlcs) + len(outgoingHtlcs)
+	numHtlcs := 0
```

Comment:
Yeah afaict, this value is only used for logging. Just define this via an expression immediately before it is used.

Reply:
cool removed